src/nxdoc/nuxeo-add-ons/nuxeo-aspera-connector/nuxeo-aspera-upgrade-notes.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Nuxeo Aspera Upgrade Notes
+description: Upgrade notes for Nuxeo Aspera.
+tree_item_index: 200
+review:
+  comment: ''
+  date: '2020-09-10'
+  status: ok
+toc: true
+---
+
+
+{{! excerpt}}
+This chapter highlights some major information about upgrade from Nuxeo Aspera Connector 2.x to Nuxeo Aspera Connector 3.0.x.
+For a listing of improvements and functional enhancements in version 3.0.x, please see the [release notes]() {{! /excerpt}}
+
+{{#> callout type='warning'}}
+Because of the changes to parameters in the nuxeo.conf file, this release is labeled as a major release.  Transfers that exist in your system **that were started/created with a previous version will not work with this release and attempts to complete them will fail with this realse**.
```

Comment:
```suggestion
Because of the changes to parameters in the **nuxeo.conf** file and also to some API's  deprecation and major changes, this release is labeled as a major release.  Transfers that exist in your system **that were started/created with a previous version will not work with this release and attempts to complete them will fail with this release**. It is recommended to don't update the Connector version with _ongoing_ transfers.
```

Reply:
I like your suggestion, but need to change the wording just a bit so can't commit suggestion directly 👍